tcii18n/translator.py

```diff
@@ -0,0 +1,15 @@
+import csv
+
+
+class Translator:
+    def __init__(self, filename):
+        self.reader = open(filename, 'r')
```

Comment:
you don't need `reader` as a class property. `reader = …` without self will be enough.